misc/N and its Double.c

```diff
@@ -0,0 +1,46 @@
+int main()
+{
+    int N,a,i,j,count=0;
+    printf("Enter N=");
+    scanf("%d",&N);
+    if(N>=2 & N<=500)
```

Comment:
As I mentioned above, u can assume that `N` lies in the range given in constraints so u do not need this condition